report_thesis/src/sections/background/linear_and_regularization_models/rigde_regression.tex

```diff
@@ -0,0 +1,20 @@
+\subsubsection{Ridge Regression}
+A major limitation of \gls{olsr} is that it cannot handle multicollinearity effectively.
+We now provide an overview of ridge regression, an extension of \gls{olsr} designed to address this limitation, based on \citet{James2023AnIS}.
+Ridge regression introduces a regularization term to the \gls{olsr} objective function to prevent overfitting and reduce the model's variance.
+This regularization term is known as the $L_2$ norm and is defined as the sum of the squared regression coefficients:
+
+$$
+L_2 = \sum_{j=1}^{p} \beta_j^2,
+$$
+
+where $\beta_j$ are the regression coefficients, and $p$ is the number of features.
+The ridge regression objective function is defined as:
+
+$$
+\text{RSS} + \lambda L_2
```

Comment:
Feels like there needs to be an objective function equal to this.
Perhaps something like:
```suggestion
\text{Obj}_{ridge} = \text{RSS} + \lambda L_2
```